wiki/_Sidebar.md

```diff
@@ -25,6 +25,9 @@
   * Testing
     * [Oppia Android Testing](https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing)
     * [End to End Testing Guide](https://github.com/oppia/oppia-android/wiki/End-to-End-Testing-Guide)
+  * Code Coverage
```

Comment:
These new pages seem to be a better fit under 'testing' I think.

Reply:
Yeah, moved them Testing.